src/bidiMapper/modules/cdp/CdpTarget.ts

```diff
@@ -133,9 +143,16 @@ export class CdpTarget {
    */
   async #unblock() {
     try {
+      await this.#cdpClient.sendCommand('Page.enable');
```

Comment:
this is very likely not correct, you need to schedule the commands at the same time, as they might not reach the renderer while they are blocked, resulting in a deadlock

Reply:
right, fixed